### PR TITLE
Added LanCache Litmus container

### PR DESCRIPTION
**Pull request by Crabbey**

### CODE REVIEW

File: lancache-litmus/overlay/hooks/entrypoint-pre.d/10_generate_litmus_statusjs.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+			done
+		done
+		echo "		]" >> $OUTPUTFILE
+		echo "	}," >> $OUTPUTFILE
+	done
+done
+echo "};" >> $OUTPUTFILE
+if [[ -d .git ]]; then
+	giturl=$(git config --get remote.origin.url)
+	echo "var urltext='Running with hostlist from <a target=\"_blank\" href=\"${giturl}\">${giturl}</a>';" >> $OUTPUTFILE;
```
Comment (VibroAxe) [inline lancache-litmus/overlay/hooks/entrypoint-pre.d/10_generate_litmus_statusjs.sh]: Can we add the current date & time to these hostlists so we can see when the list was last updated?

File: lancache-litmus/overlay/etc/nginx/sites-available/litmus.conf.d/10_litmus.conf [hunk 2]
```diff
@@ -0,0 +1,3 @@
+server_name lancache LITMUS_HOSTNAME;
```
Comment (VibroAxe) [inline lancache-litmus/overlay/etc/nginx/sites-available/litmus.conf.d/10_litmus.conf]: can we set this to default

File: lancache-litmus/overlay/hooks/entrypoint-pre.d/10_generate_litmus_statusjs.sh [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+else
+	echo "var urltext='Running with an external hostlist that is not git controlled';" >> $OUTPUTFILE;
```
Comment (VibroAxe) [inline lancache-litmus/overlay/hooks/entrypoint-pre.d/10_generate_litmus_statusjs.sh]: as above

**Review by VibroAxe — COMMENTED** [review]
Pretty good start, few suggestions:

File: lancache-litmus/overlay/etc/nginx/sites-available/litmus.conf.d/10_litmus.conf (hunk 2 above)
Comment (Crabbey) [reply lancache-litmus/overlay/etc/nginx/sites-available/litmus.conf.d/10_litmus.conf]: LITMUS_HOSTNAME defaults to lancache.lan from the Dockerfile

**Comment by VibroAxe** [comment]
lgtm as a starter, well done @Crabbey

File: lancache-litmus/overlay/etc/nginx/sites-available/10_litmus.conf [hunk 4]
```diff
@@ -0,0 +1,9 @@
+server {
+  listen 80;
```
Comment (VibroAxe) [inline lancache-litmus/overlay/etc/nginx/sites-available/10_litmus.conf]: listen 80 default;

**Pull request merged**